tests/model_fields/models.py

```diff
@@ -387,3 +387,26 @@ class UUIDChild(PrimaryKeyUUIDModel):
 
 class UUIDGrandchild(UUIDChild):
     pass
+
+###############################################################################
+# This model is used for Readonly test cases
```

Comment:
I feel like this comment is redundant given the model names. None of the other models in the file have such comments 😄